version3/js/.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 100,
+  "trailingComma": "all"
```

Comment:
Using `  "trailingComma": "all"` is not a good idea when the .js codes is not run through any transpiler, as it may lead to syntax that not every runtime accepts. Will update.

Reply:
done